custom-service-adapter/src/main/java/io/knotx/tutorials/CustomServiceAdapter.java

```diff
@@ -0,0 +1,47 @@
+package io.knotx.tutorials;
+
+import io.knotx.proxy.AdapterProxy;
+import io.knotx.tutorials.impl.CustomServiceAdapterProxyImpl;
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.Context;
+import io.vertx.core.Vertx;
+import io.vertx.core.eventbus.MessageConsumer;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import io.vertx.rxjava.ext.jdbc.JDBCClient;
+import io.vertx.serviceproxy.ProxyHelper;
+
+public class CustomServiceAdapter extends AbstractVerticle {
```

Comment:
Make more meaningful name of the adapter, e.g.: BooksDbAdapter -> as we're going to adapt database with books to knotx. (not service)
Same for configuration class, it should have name of `BooksDbAdapaterConfiguration`